addons/color-inputs/userscript.js

```diff
@@ -0,0 +1,42 @@
+export default async function ({ addon, msg, global, console }) {
+  while (true) {
+    const swatchRow = await addon.tab.waitForElement('div[class*="color-picker_swatch-row"]', {
+      markAsSeen: true,
+      reduxCondition: (state) => state.scratchGui.editorTab.activeTabIndex === 1 && !state.scratchGui.mode.isPlayerOnly,
+    });
+    const colorPicker = swatchRow.parentElement;
+    const internalKey = addon.tab.traps.getInternalKey(colorPicker);
+    const proto = colorPicker[internalKey].return.return.return.stateNode;
+    const state = colorPicker[internalKey].return.return.stateNode;
+    const headers = colorPicker.querySelectorAll("[class*='color-picker_row-header']");
+    const color = ["hue", "saturation", "brightness"];
+    color.forEach((c, i) => {
+      const header = headers[i];
+      header.style.display = "flex";
+      header.style.alignItems = "center";
+      header.style.justifyContent = "space-between";
+      header.querySelector("[class*='color-picker_label-readout']").style.display = "none";
+      const input = document.createElement("input");
+      input.type = "number";
+      input.value = Math.round(state.props[c] * 10) / 10;
+      input.min = 0;
+      input.max = 100;
+      input.className = addon.tab.scratchClass("input_input-form", "input_input-small", "input_input-small-range");
```

Comment:
You should probably get the classes from the outline width input instead of using `scratchClass` - it returns scratch-gui classes, which look slightly different from the scratch-paint equivalents.

Reply:
@mxmou Could you fix this?

Reply:
Using gui classes might actually be appropriate: https://github.com/ScratchAddons/ScratchAddons/issues/4106#issuecomment-1019356218

Reply:
Okay, keeping gui classes for now